Assets/UI/civ6common.lua

```diff
@@ -611,6 +614,7 @@ function GetLeaderUniqueTraits( leaderType:string )
 
   local unique_abilities = {};
   for i, trait in ipairs(traits) do
+    print(trait.InternalOnly);
```

Comment:
I cleaned all the print that were not necessary in the beta branch before merging, and this one was part of it. So you should remove it. 
Generally speaking only print if there is an error (so that it does not clutter the debug files)